po/fr.po

```diff
@@ -2007,7 +2003,6 @@ msgid "Internal parse error"
 msgstr "Erreur d'analyse interne"
 
 #: rust/libnewsboat/src/fslock.rs:57
-#, fuzzy
 msgid "Failed to open lock file '%s': %s"
 msgstr "Echec d'ouverture du fichier de verrouillage : '%s' (%s)"
```

Comment:
Please check this translation; the punctuation at the end of the messages is different.